src/main/java/org/broadinstitute/consent/http/models/DataAccessRequestData.java

```diff
@@ -2,92 +2,103 @@
-    Boolean oneGender;
-    String gender;
-    Boolean pediatric;
-    @SerializedName(value = "illegalbehave", alternate = "illegalBehavior")
-    Boolean illegalBehavior;
-    Boolean addiction;
-    @SerializedName(value = "sexualdiseases", alternate = "sexualDiseases")
-    Boolean sexualDiseases;
-    @SerializedName(value = "stigmatizediseases", alternate = "stigmatizedDiseases")
-    Boolean stigmatizedDiseases;
-    @SerializedName(value = "vulnerablepop", alternate = "vulnerablePopulation")
-    Boolean vulnerablePopulation;
-    @SerializedName(value = "popmigration", alternate = "populationMigration")
-    Boolean populationMigration;
-    @SerializedName(value = "psychtraits", alternate = "psychiatricTraits")
-    Boolean psychiatricTraits;
-    @SerializedName(value = "nothealth", alternate = "notHealth")
-    Boolean notHealth;
-    Boolean hmb;
-    String status;
-    Boolean poa;
-    List<DatasetEntry> datasets;
-    @SerializedName(value = "dar_code", alternate = "darCode")
-    String darCode;
-    @SerializedName(value = "partial_dar_code", alternate = "partialDarCode")
-    String partialDarCode;
-    Object restriction;
-    @SerializedName(value = "valid_restriction", alternate = "validRestriction")
-    Boolean validRestriction;
-    String translatedUseRestriction;
-    Long createDate;
-    Long sortDate;
-    List<Integer> datasetId;
-    List<DatasetDetailEntry> datasetDetail;
-
-    public DataAccessRequestData() {
-    }
+    private String referenceId;
+    private String investigator;
+    private String institution;
+    private String department;
+    private String division;
+    private String address1;
+    private String address2;
+    private String city;
+    @SerializedName(value = "zipCode", alternate = "zipcode")
+    private String zipCode;
+    private String state;
+    private String country;
+    private String projectTitle;
+    private Boolean checkCollaborator;
+    private String researcher;
+    private Integer userId;
+    private String isThePi;
+    private String havePi;
+    private String piEmail;
+    private String profileName;
+    private String pubmedId;
+    private String scientificUrl;
+    private String urlDAA;
+    private String nameDAA;
+    private Boolean eraExpiration;
+    private String academicEmail;
+    private Boolean eraAuthorized;
+    private String nihUsername;
+    private String linkedIn;
+    private String orcid;
+    private String researcherGate;
+
+    private String rus;
+    @SerializedName(value = "nonTechRus", alternate = "non_tech_rus")
+    private String nonTechRus;
+    private Boolean diseases;
+    private Boolean methods;
+    private Boolean controls;
+    private Boolean population;
+    private Boolean other;
+    private String otherText;
+    private List<OntologyEntry> ontologies;
+    private Boolean forProfit;
+    @SerializedName(value = "oneGender", alternate = "onegender")
```

Comment:
Nothing wrong here, just wanted to mention that I support these new variable names and would like to do the same on the front end.

Reply:
Yes, this is where I think I may have introduced a bug in the UI. If the `darInfo` object is expecting `onegender` vs `oneGender` (same concern with some other fields), that field may never populate. It should be an easy fix since I refactored everything to look at the same `darInfo` obhect a while back.